config/tests/samples/create/samples.go

```diff
@@ -147,16 +159,16 @@ func WaitForReady(h *Harness, unstructs ...*unstructured.Unstructured) {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			waitForReadySingleResource(h, u)
+			waitForReadySingleResource(h, u.GroupVersionKind(), k8s.GetNamespacedName(u))
 		}()
 	}
 	wg.Wait()
 }
 
-func waitForReadySingleResource(t *Harness, u *unstructured.Unstructured) {
+func waitForReadySingleResource(t *Harness, gvk schema.GroupVersionKind, id types.NamespacedName) {
```

Comment:
not a biggie but we could've kept the signature the same and gotten the namespaced name from the unstructured. Is this a better pattern ?